run.bat

```diff
@@ -0,0 +1,37 @@
+@Echo on
+set JAVA_HOME=C:\Java\jdk1.6.0_45
```

Comment:
Hi @mingqin1,

Thanks for this contribution. I don't know much about Windows development, so please forgive me in advance for the newbie question.

Is it possible in a Windows environment that these variables (JAVA_HOME, MAVEN_HOME, etc.) be already set? And if yes, would there be a way for this script to honor the already-set variables instead of clobbering them? Given that these are hard-coded paths, and that they look non-standard, I would imagine it to be better not to make assumptions on where Java or Maven is installed.

What do you think?

Thanks.

-F